main.go

```diff
@@ -296,6 +297,23 @@ func collectFilesToDeploy(absDeployPth string, config Config, tmpDir string) (fi
 	return filesToDeploy, nil
 }
 
+func compressAsTar(sourceDirPath, destinationTarPath string, isContentOnly bool) error {
```

Comment:
If we are not compressing the tar, isn't it a bit misleading to call this `compress` everywhere? I think the correct term for this whole thing is to create an archive (BTW tar is Tape ARchive 😉 )